<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление научными исследованиями по тематическому плану</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>11</y>
      <width>541</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="add_widget" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>20</y>
      <width>531</width>
      <height>631</height>
     </rect>
    </property>
    <widget class="QWidget" name="Tp_fv_add_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>531</width>
       <height>601</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="Tp_fv_add_Code_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_fv_add_VUZshortname_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_fv_add_plannedFinancing_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_fv_add_actualFinancing_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Код</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сокращенное название ВУЗа</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Плановое финансирование</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Фактическое финансирование</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Tp_fv_add_widget_saveButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_widget_close_pushButton4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="grntirub_add_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>531</width>
       <height>601</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="grnti_add_categoryCode_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="grnti_add_categoryName_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Код рубрики</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Название рубрики</string>
      </property>
     </widget>
     <widget class="QPushButton" name="grnti_add_widget_saveButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_widget_close_pushButton3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tp_nir_add_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>531</width>
       <height>601</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="Tp_nir_add_VUZcode_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_nir_add_grntiNumber_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="Tp_nir_add_grntiNature_comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_nir_add_VUZshortname_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_nir_add_grntiHead_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_nir_add_grntiCode_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_nir_add_grntiName_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_nir_add_grntiHeadPost_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Tp_nir_add_plannedFinancing_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Tp_nir_add_widget_saveButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_widget_close_pushButton2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Код ВУЗа</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Номер НИР</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Характер НИР</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сокращенное название ВУЗа</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Руководитель НИР</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Коды ГРНТИ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Название НИР</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>310</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Должность руководителя НИР</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>350</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Плановое финансирование НИР</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="VUZ_add_widget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>531</width>
       <height>601</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="VUZ_add_VUZcode_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="VUZ_add_VUZname_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string> Код ВУЗа</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>70</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Наименование ВУЗа</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Полное название ВУЗа</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="VUZ_add_VUZfullname_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="VUZ_add_VUZshortname_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="VUZ_add_region_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="VUZ_add_city_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>150</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сокращенное название ВУЗа</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Регион</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>230</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Город</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="VUZ_add_status_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="VUZ_add_areacode_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Статус</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>310</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Код области</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="VUZ_add_area_textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>350</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Область</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>390</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Тип учебного заведения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>430</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Профиль ВУЗа</string>
      </property>
     </widget>
     <widget class="QComboBox" name="VUZ_add_VUZtype_comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="VUZ_add_profile_comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="VUZ_add_widget_saveButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="add_widget_close_pushButton1">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>480</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменить</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="add_tableChoice_comboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Выберете таблицу</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_widget_reload_pushButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>Отображение таблиц</string>
    </property>
    <addaction name="action_show_VUZ"/>
    <addaction name="separator"/>
    <addaction name="action_show_Tp_nir"/>
    <addaction name="separator"/>
    <addaction name="action_show_grntirub"/>
    <addaction name="separator"/>
    <addaction name="action_show_Tp_fv"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Работа с бд</string>
    </property>
    <addaction name="action_check"/>
    <addaction name="separator"/>
    <addaction name="action_add"/>
    <addaction name="action_6"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="add_cod_grnti"/>
    <addaction name="separator"/>
    <addaction name="filter_grnti"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Формирование отчетности</string>
    </property>
    <addaction name="action_9"/>
    <addaction name="action_10"/>
   </widget>
   <addaction name="menu_1"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_show_VUZ">
   <property name="text">
    <string>VUZ</string>
   </property>
  </action>
  <action name="action_show_Tp_nir">
   <property name="text">
    <string>Tp_nir</string>
   </property>
  </action>
  <action name="action_show_grntirub">
   <property name="text">
    <string>grntirub</string>
   </property>
  </action>
  <action name="action_check">
   <property name="text">
    <string>Проверка целостности</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>Добавление записи</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Редактирование</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Удаление запси</string>
   </property>
  </action>
  <action name="action_show_Tp_fv">
   <property name="text">
    <string>Tp_fv</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Формирование документов</string>
   </property>
  </action>
  <action name="action_10">
   <property name="text">
    <string>Сохранение условий фильтрации</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Сформировать распоряжение</string>
   </property>
  </action>
  <action name="add_cod_grnti">
   <property name="text">
    <string>Добавление кода грнти</string>
   </property>
  </action>
  <action name="filter_grnti">
   <property name="text">
    <string>Фильтрация по коду ГРНТИ</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
